openeo/rest/connection.py

```diff
@@ -1065,6 +1065,37 @@ def describe_process(self, id: str, namespace: Optional[str] = None) -> dict:
 
         raise OpenEoClientException("Process does not exist.")
 
+    """
+        Loads all available processes of the back end.
+
+        :param namespace: The namespace for which to list processes.
+
+        :return: Dictionary of All available processes of the back end.
+    """
+
+    def dict_processes(self, namespace: Optional[str] = None) -> dict:
```

Comment:
I don't think it's necessary to implement this method. We already have `list_processes` and `describe_process`, which should cover most/all needs (including the one in this PR)